alternatives/colormatic.mjs

```diff
@@ -6,10 +6,10 @@ export default new Mod(
 	"An independent implementation of the custom colors mod for Minecraft.",
 )
 .icon("https://cdn.modrinth.com//data/V4IQxkZC/icon.png")
-.add_version({ loader: ["fabric", "quilt"], v: [14, 15, 16, 17, 18, 19] })
+.add_version({ loader: ["quilt"], v: [19] }, { loader: ["fabric"], v: [14, 15, 16, 17, 18, 19] })
```

Comment:
I don't remember of any compatibility being reported for colormatic with Quilt on 1.18.

```suggestion
.add_version({ loader: ["quilt"], v: [18, 19] }, { loader: ["fabric"], v: [14, 15, 16, 17, 18, 19] })
```